### PR TITLE
 net-misc/streamlink: version bump to 0.9.0

### PR DESCRIPTION
**Pull request by soredake**

### CODE REVIEW

File: net-misc/streamlink/streamlink-0.8.1.ebuild [hunk 1]
```diff
@@ -24,6 +24,7 @@ RDEPEND="dev-python/pycryptodome[${PYTHON_USEDEP}]
 	dev-python/backports-shutil_which[$(python_gen_usedep 'python2*')]
 	dev-python/backports-shutil_get_terminal_size[$(python_gen_usedep 'python2*')]
 	dev-python/pycountry[${PYTHON_USEDEP}]
+	dev-python/websocket-client[${PYTHON_USEDEP}]
```
Comment (SoapGentoo) [inline net-misc/streamlink/streamlink-0.8.1.ebuild]: please revbump this ebuild (due to the required change to the VDB)

Comment (soredake) [reply net-misc/streamlink/streamlink-0.8.1.ebuild]: done

**Comment by soredake** [comment]
Ping.

**Comment by soredake** [comment]
ping @SoapGentoo

**Comment by soredake** [comment]
ping @mgorny

**Comment by holgersson32644** [comment]
Hi, could you please
* revbump streamlink-0.8.1.ebuild to -r1 or delete it (as you bump to 0.9.0)
* rebase your commit after that?

File: net-misc/streamlink/streamlink-0.9.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	virtual/python-singledispatch[${PYTHON_USEDEP}]
+	dev-python/backports-shutil_which[$(python_gen_usedep 'python2*')]
+	dev-python/backports-shutil_get_terminal_size[$(python_gen_usedep 'python2*')]
+	dev-python/pycountry[${PYTHON_USEDEP}]
+	dev-python/websocket-client[${PYTHON_USEDEP}]
+	media-video/rtmpdump
+	virtual/ffmpeg"
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]
+	doc? ( dev-python/sphinx[${PYTHON_USEDEP}]
+		dev-python/docutils[${PYTHON_USEDEP}] )
```
Comment (SoapGentoo) [inline net-misc/streamlink/streamlink-0.9.0.ebuild]:
```
doc? (
	dev-python/sphinx[${PYTHON_USEDEP}]
	dev-python/docutils[${PYTHON_USEDEP}]
)
```

File: net-misc/streamlink/streamlink-0.9.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+	test? ( dev-python/mock[$(python_gen_usedep 'python2*')]
+		${RDEPEND} )"
```
Comment (SoapGentoo) [inline net-misc/streamlink/streamlink-0.9.0.ebuild]: here too

File: net-misc/streamlink/streamlink-0.9.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+LICENSE="BSD-2 Apache-2.0"
+SLOT="0"
+IUSE="doc test"
+
+RDEPEND="dev-python/pycryptodome[${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]
+	virtual/python-futures[${PYTHON_USEDEP}]
+	virtual/python-singledispatch[${PYTHON_USEDEP}]
+	dev-python/backports-shutil_which[$(python_gen_usedep 'python2*')]
+	dev-python/backports-shutil_get_terminal_size[$(python_gen_usedep 'python2*')]
```
Comment (SoapGentoo) [inline net-misc/streamlink/streamlink-0.9.0.ebuild]: you probably want `$(python_gen_cond_dep dev-python/backports-shutil_get_terminal_size[${PYTHON_USEDEP}] 'python2*')` here

**Comment by soredake** [comment]
@holgersson32644 @SoapGentoo done

File: net-misc/streamlink/streamlink-9999.ebuild [hunk 5]
```diff
@@ -22,15 +22,20 @@ RDEPEND="dev-python/pycryptodome[${PYTHON_USEDEP}]
...
+	dev-python/websocket-client[${PYTHON_USEDEP}]
 	media-video/rtmpdump
 	virtual/ffmpeg"
 DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]
-	doc? ( dev-python/sphinx[${PYTHON_USEDEP}]
-		dev-python/docutils[${PYTHON_USEDEP}] )
-	test? ( dev-python/mock[$(python_gen_usedep 'python2*')]
-		${RDEPEND} )"
+	doc? (
+			dev-python/sphinx[${PYTHON_USEDEP}]
```
Comment (SoapGentoo) [inline net-misc/streamlink/streamlink-9999.ebuild]: too much indentation

File: net-misc/streamlink/streamlink-9999.ebuild [hunk 6]
```diff
@@ -22,15 +22,20 @@ RDEPEND="dev-python/pycryptodome[${PYTHON_USEDEP}]
...
+			dev-python/docutils[${PYTHON_USEDEP}]
+	)
+	test? (
+			dev-python/mock[$(python_gen_usedep 'python2*')]
```
Comment (SoapGentoo) [inline net-misc/streamlink/streamlink-9999.ebuild]: ofc do the `python_gen_cond_dep` thing here too

**Comment by soredake** [comment]
@SoapGentoo done

**Comment by SoapGentoo** [comment]
@soredake I slightly changed your commitmsg to be in line with GLEP 66

File: net-misc/streamlink/streamlink-9999.ebuild [hunk 7]
```diff
@@ -22,15 +22,20 @@ RDEPEND="dev-python/pycryptodome[${PYTHON_USEDEP}]
 	virtual/python-futures[${PYTHON_USEDEP}]
 	virtual/python-singledispatch[${PYTHON_USEDEP}]
 	dev-python/backports-shutil_which[$(python_gen_usedep 'python2*')]
```
Comment (CrazyTushkan) [inline net-misc/streamlink/streamlink-9999.ebuild]: @soredake, can you please clarify for me why didn't you change `python_gen_usedep` to `python_gen_cond_dep` also here?

**Pull request closed without merging**